### ISSUE TITLE
Document steps to run WRF/METplus/Visualization containers together

### ISSUE THREAD

**Issue by georgemccabe** [NCAR/i-wrf#6]
Documentation exists to run WRF through a container and to utilize multiple containers, but all of the necessary information needed to run what we need is not clearly documented in once place.

## Describe the Task ##
Determine steps needed to run these tools via containers and document the process so it is easier for new users to accomplish.

### Time Estimate ###
*Estimate the amount of work required here.*
*Issues should represent approximately 1 to 3 days of work.*

### Sub-Issues ###
Consider breaking the task down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
*List relevant project deadlines here or state NONE.*

### Funding Source ###
*Define the source of funding and account keys here or state NONE.*

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [ ] Select **component(s)**
- [ ] Select **priority**
- [ ] Select **requestor(s)**

### Projects and Milestone ###
- [ ] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED PROJECT ASSIGNMENT** label
- [ ] Select **Milestone** as the next official version or **Future Versions**

## Task Checklist ##
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issues
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

**Comment by georgemccabe** [comment]
Melissa Bukovsky documented the process of running the containers using singularity including notes about issues encountered.

Link to Google Doc: [IWRF Container Notes](https://docs.google.com/document/d/1bFgmE8yP-FDkoShA4A7jd6Uk3OVoEfF0LxL9R5wPTMM/edit)

**Comment by georgemccabe** [comment]
Notes from @rcplane running containers: [IWRF_Container_Notes_Cornell_CAC_Red_Cloud_First_Run_2023_05](https://docs.google.com/document/d/1hVg5hvDT41jzvcOz_6YoLGGsk04D2WT0xpG-gRimsAw/edit#heading=h.jklogef7juk5)

**Comment by JohnHalleyGotway** [comment]
Update from March 15, 2024 meeting.

- [x] David Hahn has already used the I-WRF system to run the WRF tutorial on Cheyenne.
- [ ] @rcplane is adapting instructions from David Hahn to run the WRF tutorial case on Derecho.
- [ ] @rcplane will document these steps in whatever way he'd like and hand off the result to @JohnHalleyGotway.
- [ ] @JohnHalleyGotway will test/replicate these Derecho instructions and document them via the I-WRF RTD page.
- [ ] @jaredalee will re-test the resulting instructions in RTD.

**Comment by georgemccabe** [comment]
@hahnd created instructions to run WRF for the Hurricane Matthew use case on Derecho in PR #62. I moved these instructions to the new format for use cases and modified the instructions to more closely match the instructions for other platforms. This issue can be closed once PR #62 is merged.
